src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -996,40 +989,13 @@ public enum JsonUnknownDerivedTypeHandling
     {
         FailSerialization = 0,
         FallBackToBaseType = 1,
-        FallBackToNearestAncestor = 2
+        FallBackToNearestAncestor = 2,
     }
     public enum JsonUnknownTypeHandling
     {
         JsonElement = 0,
         JsonNode = 1,
     }
-    public partial class JsonPolymorphicTypeConfiguration : System.Collections.Generic.ICollection<(System.Type DerivedType, object? TypeDiscriminator)>, System.Collections.Generic.IEnumerable<(System.Type DerivedType, object? TypeDiscriminator)>, System.Collections.IEnumerable
```

Comment:
First change made in the implementation of this PR: Remove `JsonPolymorphicTypeConfiguration`